README.md

```diff
@@ -56,6 +56,7 @@ The main Docker repo URL, common between Debian and RHEL systems.
     docker_apt_repository: "deb [arch={{ docker_apt_arch }}] {{ docker_repo_url }}/{{ ansible_distribution | lower }} {{ ansible_distribution_release }} {{ docker_apt_release_channel }}"
     docker_apt_ignore_key_error: True
     docker_apt_gpg_key: "{{ docker_repo_url }}/{{ ansible_distribution | lower }}/gpg"
+    docker_apt_filename: omit
```

Comment:
For this option, could you just put instead of `omit`, `custom-filename-here-if-desired` or something that would be a more practical example. Or maybe we could leave it empty in `defaults/main.yml` instead, that might be preferable (`docker_apt_filename: ''`, and then document that value here, so it is more obvious it should be a string.

Reply:
Agree that the latter is probably preferable. Unfortunately, it seems like you can't give an empty string for that argument. This PR fail CI, but #369, which has a non-empty string, passes CI.

We could hack around it with something like `filename: {{ docker_apt_filename if docker_apt_filename != "" else omit }}`, assuming that works.

Alternatively, it may make more sense to just close this in favor of #369, if you plan on releasing 7.0.0 soon. This PR is mainly intended as a temporary fix until the defaults get updated anyway.